app/gather/assets/apps/utils/paths.jsx

```diff
@@ -106,6 +106,15 @@ export const getEntitiesAPIPath = ({ id, ...params }) => {
   return buildAPIPath(KERNEL_APP, 'entities', id, params)
 }
 
+/**
+ * Returns the API url to get the list of topics by Survey (Mapping)
+ *
+ * @param {number}  id          - Mapping id
```

Comment:
Actually, it's not a number but an `uuid` and should change the other ones too :woman_facepalming: